### DISCUSSION TITLE
Quantification of quartz CPO matching and fabric strength via fibreVolume

### DISCUSSION THREAD

**Discussion by orlandinebsd** [mtex-toolbox/mtex#1196] [Ask Anything]
Hello, all! 

I have been squinting at pole figures of Scottish quartz for the last few weeks, trying to interpret slip system patterns _a la_ [Zuo et al 2021](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2020TC006514) Figure 3:

![Zuoetal2021Fig3](https://user-images.githubusercontent.com/90639180/133129285-f3e7bcd4-a865-4991-969e-9b293c85efbd.PNG)

While it was fun at first, I have become gradually disgusted with my human brain's qualitative approach. It seems like these theoretical pole figures could be approximated very well by MTEX fibres, and then I could just throw my messy natural data at them to see which has the highest % match via fibreVolume. I say that not really having every worked with 'fibres' in MTEX, but _surely_ this would work! 

In fact, I'm pretty certain that that this isn't even an original thought and at some point in the last year or two I read about someone doing exactly this with either quartz or olivine. Of course when I read it, I thought 'gosh that sounds like it was a lot of work' and then promptly forgot everything about the publication. Does anyone here happen to know of any such publications? Or better yet, has anyone already defined these unchanging theoretical fibres and be willing to post them here on github for the betterment of the entire geoscience EBSD community? That way we could skip the slapfights over how exactly an endmember fibre is defined by just all using the same models.

**Comment by zmichels** [comment]
Hmmm. It sounds like a great project. 

In addition to (or perhaps instead of) your fiber approach, you might try incorporating/combining an analysis of low-angle (intragranular)  neighbor-neighbor (correlated/innerBoundary) misorientation axes (example of outer/grain boundary misorientations on this page: https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html) and some of the analyses mentioned on this page about Schmid Factor and slip system analysis (https://mtex-toolbox.github.io/SchmidtFactor.html)… to constrain the potential activity of different slip systems.

Cheers,
Zach

**Reply by orlandinebsd** [reply]
Hello, Zach! 

Yes, I have been greatly enjoying the new functionality of innerBoundary with this project, but there is reason to suspect that the LAGB misorientation axis (assuming only tilt) is reflecting a younger, incomplete stage of deformation that was able to generate a new suite of subgrain boundaries but not overprint the high-angle grain boundary network formed at higher temperatures. 

The Schmid Factor is a great idea, and could definitely help prop up a squidgy pole figure interpretation! Misorientation plots could also help, but I might need to make another post concerning calcMisorientation's apparent refusal to handle grain.meanOrientation input. The way that function seems to be described, it is only comparing pixel orientations and not grain orientations?

**Comment by zmichels** [comment]
One possible workaround for analyzing uncorrelated misorientations using the mean orientations of grains might be to compute a Misorientation Distribution Function (MDF) from an ODF that you construct using the mean orientations. 

Maybe something along the lines of:

% specify phase of interest
phase = 'q';
 
% Assuing already have computed grains... 
% get the mean orientation of grains of the phase
o = grains(phase).meanOrientation;
 
% Compute an ODF
odf = calcDensity(o);
 
% Compute an MDF
mdf = calcMDF(odf);
 
% make some plots...
% misorientation axes
figure,
plotAxisDistribution(mdf);
 
% misorientation angles
figure,
plotAngleDistribution(mdf)



Z

**Reply by orlandinebsd** [reply]
Hi Zach! 

I had not thought of using calcMDF in this way, and it does allow calculating these distributions either from the ODF or directly from the grains.meanOrientation! Thank you! This raises more questions for me, but I will start them in another topic. I will leave this one open for a while longer just in case someone has already made the quartz fibres and is willing to spare me re-inventing that particular wheel.

**Reply by filippeof** [reply]
Hi Phil,
I will try to find a way to subscribe to your questions. Always something interesting coming up!

I think @DavidMainprice worked with something like this.
You may try to contact him or start with this presentation (e.g. p.49):
http://www.gm.univ-montp2.fr/PERSO/mainprice/W_data/My_Lectures/MTEX_Geosciences_Examples_EBSD_2014_Pittsburgh.pdf

Cheers,
Filippe
